config/locales/models/mission.en.yml

```diff
@@ -20,6 +20,8 @@ en:
     errors:
       models:
         mission:
+          messages:
+            insufficient_proficiency: Your cash register proficiency is insufficient to take the last slot of time from %{start_time.strftime('%Hh%M')} to %{(start_time + 90.minutes).strftime('%Hh%M')}
```

Comment:
Your proficiency on the cash register is insufficient to enroll on this last slot available (from xxx to xxx), as we need at least one person that satisfies the required cash register proficiency level.